site/en/blog/new-in-devtools-101/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: "layouts/blog-post.njk"
+title: "What's New In DevTools (Chrome 101)"
+authors:
+  - jecelynyeen
+date: 2022-04-12
+updated: 2022-04-12
+description: "Import and export user flow as JSON, support hwb() colors, view cascade layers in the Styles pane and more."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/mPXyPxgzu9PAoAgctnn1.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-101
+---
+
+<!-- image/dPDCek3EhZgLQPGtEG3y0fTn4v82/B5K95d3OAmyytzJVCyr1.jpg -->
+
+{% include 'partials/devtools/en/banner.md' %}
+
+
+## Import and export recorded user flows as a JSON file  {: #recorder }
+
+The [Recorder](/docs/devtools/recorder) panel now supports importing and exporting user flow recordings as a JSON file. This addition makes it easier to share user flows and can be useful for bug reporting.
+
+For example, download this [JSON file](https://storage.googleapis.com/web-dev-uploads/file/dPDCek3EhZgLQPGtEG3y0fTn4v82/vzQbv2rUfTz2DEmx06Gv.json). You can import it with the import button and [replay the user flow](/docs/devtools/recorder/#replay).
+
+{% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/Jy7NEDZs6XJb90EWqETj.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
+
+Apart from that, you can export the recording as well. After [recording a user flow](/docs/devtools/recorder/#record), click on the export button. There are 3 export options:
+
+- **Export as a JSON file**. Download the recording as a JSON file.
+- **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script. 
+- **Export as a Puppeteer script** . Download the recording as [Puppeteer](https://pptr.dev/) script.
+
+Consult [the documentation](/docs/devtools/recorder) to learn more about the differences between these options.
```

Comment:
Neither this blog post nor the linked docs don’t explain what @puppeteer/replay is, why devs might want to use it, or why tooling authors might want to build their own integrations on top of it. Let’s document this somewhere? cc @jecfish @OrKoN

Reply:
Documenting... https://github.com/GoogleChrome/developer.chrome.com/pull/2605/files#r858357907